www/index.html

```diff
@@ -977,7 +977,22 @@
           {{/each image}}
         </div>
       </script>
-  </section>
+
+      <script type="text/template7" id="checkTermsVersionTemplate">
```

Comment:
Think this name makes more sense.
```suggestion
      <script type="text/template7" id="termsVersionTemplate">
```